functions/[[catchall]].js

```diff
@@ -0,0 +1,43 @@
+const targetDomain =
+  process.env.LARASITE_DOMAIN || 'sequence-diagram.zenuml.com';
```

Comment:
`Uncaught ReferenceError: process is not defined at functionsWorker-0.8307498740501384.js:2:20` see: https://dash.cloudflare.com/8d5fc7ce04adc5096f52485cce7d7b3d/pages/view/docs/181b9cc3-2394-4172-8a34-6d9cb42f3761